helpers.py

```diff
@@ -15,7 +18,10 @@ def get_config_value(config, name, inverter_or_template, pvinverternumber, defau
         return config[f"{inverter_or_template}{pvinverternumber}"][name]
     else:
         if defaultvalue is None:
-            raise ValueError(f"config entry '{name}' not found. Hint: Deprecated Host ONPREMISE entries must be moved to DEFAULT section")
+            raise ValueError(
+                f"config entry '{name}' not found. \
+                Hint: Deprecated Host ONPREMISE entries must be moved to DEFAULT section"
+            )
```

Comment:
Could you try to find a nicer formatting here? 2 lines should be enough if 1 is too long.

Reply:
Sure 👍